examples/scripts/example32-total-rows.js

```diff
@@ -0,0 +1,180 @@
+import React, { useCallback, useMemo, useState } from 'react';
+import ReactDataGrid from 'react-data-grid';
+import exampleWrapper from '../components/exampleWrapper';
+
+const columns = [
+  {
+    key: 'id',
+    name: 'ID',
+    frozen: true
+  },
+  {
+    key: 'task',
+    name: 'Title',
+    width: 200,
+    frozen: true,
+    formatter: ({ value, row: { selectedIndexes}, isBottomPinned }) => {
+      if (isBottomPinned) {
+        return <strong>{ selectedIndexes ? `${selectedIndexes.length} rows selected` : 'Total'}</strong>;
+      }
+
+      return <div>{value}</div>
+    }
+  },
+  {
+    key: 'priority',
+    name: 'Priority',
+    width: 200
+  },
+  {
+    key: 'issueType',
+    name: 'Issue Type',
+    width: 200
+  },
+  {
+    key: 'complete',
+    name: '% Complete',
+    width: 200
+  },
+  {
+    key: 'startDate',
+    name: 'Start Date',
+    width: 200
+  },
+  {
+    key: 'completeDate',
+    name: 'Expected Complete',
+    width: 200
+  },
+  {
+    key: 'cost',
+    name: 'Resource cost (USD)',
+    width: 200,
+    formatter: ({ value }) => <div>{ value.toLocaleString('en-US') }</div>
+  },
+  {
+    key: 'hours',
+    name: '# of working hours',
+    width: 200,
+    formatter: ({ value }) => <div>{ value.toLocaleString('en-US', { maximumFractionDigits: 1 }) }</div>
```

Comment:
```suggestion
    formatter: ({ value }) => <div>{ value }</div>
```
They're just integers, so we can simplify.